include/nuttx/wdt_notifier.h

```diff
@@ -0,0 +1,97 @@
+/****************************************************************************
+ * include/nuttx/wdt_notifier.h
```

Comment:
```suggestion
 * include/nuttx/timer/watchdog_notifier.h
```

Reply:
I moved the contents of `watchdog_notifier.h` to `timers/ watchdoer.h` and removed `watchdog_notifier.h`